### PR TITLE
[NFI]: Refactor LoadStoreOpToLLVM.cpp

### PR DESCRIPTION
**Pull request by etiotto**

### CODE REVIEW

File: third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp [hunk 1]
```diff
@@ -1459,7 +1449,8 @@ struct AtomicRMWOpConversion
       // emit unsupported feature error.
       if (valueElemNBits == 16) {
         op.emitWarning(
-            "'tt.atomic_rmw' op fp16 datatype is not supported in the target "
+            "'tt.atomic_rmw' op fp16 datatype is not supported in the "
+            "target "
```
Comment (whitneywhtsang) [inline third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp:1453]: undo or combine with the next line?

File: third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp [hunk 2]
```diff
@@ -1561,10 +1552,10 @@ struct AtomicRMWOpConversion
 
     rmwVal = bitcast(rmwVal, valueElemTy);
 
-    // Align pointer by 4 bytes by zeroing lower address bits. Atomically read
-    // a vector of two fp16 values as a single i32. The second lowest bit is
-    // extracted to later be used as an index to extract the required vector
-    // element.
+    // Align pointer by 4 bytes by zeroing lower address bits. Atomically
```
Comment (whitneywhtsang) [inline third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp:1555]: I am fine with this change, but curious why? it is needed for clang-format, why it didn't fail before?

**Review by whitneywhtsang — APPROVED** [review]

File: third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp (hunk 2 above)
Comment (etiotto) [reply third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp]: removed

File: third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp (hunk 1 above)
Comment (etiotto) [reply third_party/intel/lib/TritonIntelGPUToLLVM/LoadStoreOpToLLVM.cpp]: done

**Pull request merged**